projects/plugins/jetpack/extensions/blocks/blogroll/use-recommendations.js

```diff
@@ -2,13 +2,17 @@ import apiFetch from '@wordpress/api-fetch';
 import { useEffect, useRef, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
-export default function useRecommendations() {
+export default function useRecommendations( enabled = false ) {
 	const [ isLoading, setIsLoading ] = useState( false );
 	const [ errorMessage, setErrorMessage ] = useState( null );
 	const [ recommendations, setRecommendations ] = useState( [] );
 	const abortControllerRef = useRef();
 
 	useEffect( () => {
+		if ( ! enabled ) {
+			return;
```

Comment:
let's set `isLoading` false here. And we should pass `enabled` to deps below.

Reply:
Updated